### PR TITLE
Implement new connections page + flower garden

### PR DESCRIPTION
**Pull request by kalidiagne**

### CODE REVIEW

**Comment by stevenelleman** [comment]
Otherwise looks fine to me and works locally.

File: apps/frontend/src/pages/people/[username].tsx [hunk 1]
```diff
@@ -475,6 +565,15 @@ const UserProfilePage: React.FC = () => {
     return;
   }
 
+  let size = "sprout";
+  if (user?.userData?.connectionPSISize && user?.userData?.connectionPSISize[connection.user.username]) {
+    size = flowerSize(user?.userData?.connectionPSISize[connection.user.username]);
+  }
+
+  const flowerStage = size;
+  const flowerIndex = "2";
```
Comment (RiverRuby) [inline apps/frontend/src/pages/people/[username].tsx:574]: Needs be mod 10

File: apps/frontend/tailwind.config.ts [hunk 2]
```diff
@@ -45,6 +45,8 @@ const config: Config = {
         quaternary: "var(--color-quaternary)",
         cinnamon: "var(--color-cinnamon)",
 
+        pink: "#ff9df8",
```
Comment (RiverRuby) [inline apps/frontend/tailwind.config.ts:48]: Unideal but we can fix later

Comment (RiverRuby) [reply apps/frontend/tailwind.config.ts:48]: geneerally using variabes is really good when we want to do things like dark mode, i'd try to find the right variable here

**Review by RiverRuby — APPROVED** [review]
LGTM minus nits

**Pull request merged**